Moise1/Banka
# Applying the LFA's feedback for my UI (Challenge 1)

#### What does this PR do? 
- Refurbishing web pages to look good as required by the LFA. 

#### Description of Task to be completed?
#### _Have the following pages adjusted_ :

 - Home page (well-colored text, fixed resolution issues)
- User Profile (user's details, editable fields)
- Cashier page (accounts' info before making transactions)

#### How should this be manually tested?

- After opening this project on this gh-pages, navigate these pages to see the changes.

#### Any background context you want to provide?
None
What are the relevant pivotal tracker stories?
[###165379833](https://www.pivotaltracker.com/n/projects/2322777)

#### Screenshots (if appropriate)
1.Home page  ![Banka | Home ](https://user-images.githubusercontent.com/25052255/56208724-16589680-6052-11e9-9b13-66554576bdad.png)

2. Cashier Page ![Banka | Cashier](https://user-images.githubusercontent.com/25052255/56208814-4d2eac80-6052-11e9-9597-ac15da3e1a7d.png) 

3. Client's Profile ![Banka | My Profile ](https://user-images.githubusercontent.com/25052255/56208856-65063080-6052-11e9-81f5-b5df5885692c.png)


Questions:
